### PR TITLE
Initial PSP patching guide

### PR DESCRIPTION
**Pull request by Beyley**

### CODE REVIEW

**Comment by jvyden** [comment]
Could use an update for Vita since allefresher supports that now. Don't know if we want that as a separate guide though.

**Comment by Beyley** [comment]
Ready for review, although theres some cross-repo dependencies on here, as this guide assumes theres already been another release of Refresher (eg. a release post-Vita support), but we should wait to do that until this is merged, so that we can link to this guide from the next release of Refresher

tl;dr
lets merge this before the next Refresher release, then update Refresher to add a link to this guide, then push a new release of Refresher

File: patching/psp.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+Refresher has support for automatically deploying Allefresher onto a PSP or Vita, with the relavent configuration automatically filled out. This will let you connect to a custom server on LittleBigPlanet for the PSP (on Vita, this would be through Adrenaline).
+
+It also assumes that you've already downloaded Refresher. If you haven't, head on over to the [GitHub page](https://github.com/LittleBigRefresh/Refresher) to grab yourself a copy.
+
+![A Screenshot of Refresher's PSP patching screen](https://littlebigrefresh.github.io/Docs/patching/images/refresher-psp.png)
+
+## Step 1: Jailbreak your device and install a CFW
+
+We can't help you with this, but there are various guides out there you can go out and find. We personally recommend the guides on [pspunk](https://www.pspunk.com/psp-cfw/) and [vita.hacks.guide](https://vita.hacks.guide/), for PSP and Vita respectively.
```
Comment (jvyden) [inline patching/psp.md]:
```suggestion
We can't help you with this directly, but there are various guides out there you can go out and find. We personally recommend the guides on [pspunk](https://www.pspunk.com/psp-cfw/) and [vita.hacks.guide](https://vita.hacks.guide/) for PSP and Vita respectively.
```

File: patching/psp.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+### PSP
+
+Try to use the latest firmware, and we reccomend PRO C over other CFWs, as they have not been thoroughly tested with Allefresher yet. It is possible to dual-boot CFWs, so if you need a CFW such as ARK-4 for other homebrew, we'd reccomend dual-booting PRO C as well until Allefresher is guarenteed to work well on ARK-4.
```
Comment (jvyden) [inline patching/psp.md]:
```suggestion
Try to use the latest firmware, and we recommend PRO C over other CFWs, as they have not been thoroughly tested with Allefresher yet. It is possible to dual-boot CFWs, so if you need a CFW such as ARK-4 for other homebrew, we'd recommend dual-booting PRO C as well until Allefresher is guaranteed to work well on ARK-4.
```

File: patching/psp.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+### Vita
+
+You'll need to setup Adrenaline, and use the latest version of the Adrenaline ePSP firmware (as with PSP, ARK-4 is untested).
+
+## Step 2: Connect your PSP/Vita or Memory Stick to your computer
```
Comment (jvyden) [inline patching/psp.md:21]: Doesn't Vita require special steps to get filesystem access over USB?

**Review by jvyden — DISMISSED** [review]
Just grammar. Trusting you for PSP but I'll ask Ture to confirm Vita instructions

**Comment by jvyden** [comment]
@turecross321 Do you mind reviewing the Vita portions of these instructions for us? Are we missing anything important that should be in the guide?

File: patching/psp.md (hunk 3 above)
Comment (Beyley) [reply patching/psp.md:21]: > Doesn't Vita require special steps to get filesystem access over USB?

Not sure, im writing the vita instructions assuming the person reading it knows how to get USB to work in vitashell, as i dont know the exact steps, we should definitely get someone to write a better vita section, as i dont know here

File: patching/psp.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
+# Using Refresher to setup Allefresher on PSP/Vita
+
+Refresher has support for automatically deploying Allefresher onto a PSP or Vita, with the relavent configuration automatically filled out. This will let you connect to a custom server on LittleBigPlanet for the PSP (on Vita, this would be through Adrenaline).
+
+It also assumes that you've already downloaded Refresher. If you haven't, head on over to the [GitHub page](https://github.com/LittleBigRefresh/Refresher) to grab yourself a copy.
```
Comment (turecross321) [inline patching/psp.md]: I think replacing "it also" with "this guide" would make more sense.

File: patching/psp.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+We can't help you with this directly, but there are various guides out there you can go out and find. We personally recommend the guides on [pspunk](https://www.pspunk.com/psp-cfw/) and [vita.hacks.guide](https://vita.hacks.guide/) for PSP and Vita respectively.
+
+### PSP
+
+Try to use the latest firmware, and we recommend PRO C over other CFWs, as they have not been thoroughly tested with Allefresher yet. It is possible to dual-boot CFWs, so if you need a CFW such as ARK-4 for other homebrew, we'd recommend dual-booting PRO C as well until Allefresher is guaranteed to work well on ARK-4.
+
+### Vita
+
+You'll need to setup Adrenaline, and use the latest version of the Adrenaline ePSP firmware (as with PSP, ARK-4 is untested).
```
Comment (turecross321) [inline patching/psp.md]: Should be "set up" as opposed to "setup" since this is using it as a verb

File: patching/psp.md (hunk 3 above)
Comment (turecross321) [reply patching/psp.md:21]: Vita filesystem access is done by opening VitaShell and pressing select. The user might have to press start and switch the "SELECT button" from FTP to USB before pressing select if they were previously using FTP.

File: patching/psp.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+If you connected your PSP directly to your computer over USB, you should select USB Mode on the XMB.
+
+Mount the drive and open it in your file explorer.
+
+## Step 3: Setup Refresher
+
+Open Refresher, and select "PSP Setup".
+
+Inside Refresher you should see at a dropdown labeled "PSP Drive". In that dropdown, select the correct path. If your console's drive does not appear there, check to make sure it is mounted, and contains a `PSP` or `PSPEMU` folder on it.
```
Comment (turecross321) [inline patching/psp.md]: extremely minor nitpick, but the `pspemu` folder is lowercase on the Vita

File: patching/psp.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
+# Using Refresher to setup Allefresher on PSP/Vita
+
+Refresher has support for automatically deploying Allefresher onto a PSP or Vita, with the relavent configuration automatically filled out. This will let you connect to a custom server on LittleBigPlanet for the PSP (on Vita, this would be through Adrenaline).
```
Comment (turecross321) [inline patching/psp.md]: "relavent" should be replaced with "relevant"

File: patching/psp.md [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+
+![A Screenshot of Refresher's PSP patching screen](https://littlebigrefresh.github.io/Docs/patching/images/refresher-psp.png)
+
+## Step 1: Jailbreak your device and install a CFW
+
+We can't help you with this directly, but there are various guides out there you can go out and find. We personally recommend the guides on [pspunk](https://www.pspunk.com/psp-cfw/) and [vita.hacks.guide](https://vita.hacks.guide/) for PSP and Vita respectively.
+
+### PSP
+
+Try to use the latest firmware, and we recommend PRO C over other CFWs, as they have not been thoroughly tested with Allefresher yet. It is possible to dual-boot CFWs, so if you need a CFW such as ARK-4 for other homebrew, we'd recommend dual-booting PRO C as well until Allefresher is guaranteed to work well on ARK-4.
```
Comment (turecross321) [inline patching/psp.md]: The "and" feels a bit weird.

I think something like "Try to use the latest firmware. We recommend PRO C over other CFWs..." works better

**Review by turecross321 — COMMENTED** [review]
some small suggestions

File: patching/psp.md (hunk 5 above)
Comment (turecross321) [reply patching/psp.md]: The comma after "You'll need to setup Adrenaline" feels like it shouldnt be there to me

Comment (turecross321) [reply patching/psp.md]: i think "You'll need to set up Adrenaline with the latest version of the Adrenaline ePSP firmware" sounds better

File: patching/psp.md (hunk 6 above)
Comment (jvyden) [reply patching/psp.md]: It actually changes depending on the way you mount it - PSP/Vita's filesystem is apparently case insensitive

Comment (Beyley) [reply patching/psp.md]: Yes, when Logan was doing their testing, i saw both PSPEMU and pspemu displayed, depending on where it was being shown

File: patching/psp.md [hunk 9]
```diff
@@ -1,8 +1,8 @@
 # Using Refresher to setup Allefresher on PSP/Vita
 
-Refresher has support for automatically deploying Allefresher onto a PSP or Vita, with the relavent configuration automatically filled out. This will let you connect to a custom server on LittleBigPlanet for the PSP (on Vita, this would be through Adrenaline).
+Refresher has support for automatically deploying Allefresher onto a PSP or Vita, with the relevent configuration automatically filled out. This will let you connect to a custom server on LittleBigPlanet for the PSP (on Vita, this would be through Adrenaline).
```
Comment (turecross321) [inline patching/psp.md]: relevant* !!!

File: patching/psp.md [hunk 10]
```diff
@@ -54,7 +60,7 @@ If successful, Refresher will open a pop-up presenting the information it gather
 
 ## Step 5: Finalize the setup!
 
-When all is set and ready to go, click the patch button. It should spit out the relavent files onto your console.
+When all is set and ready to go, click the patch button. It should spit out the relevent files onto your console.
```
Comment (turecross321) [inline patching/psp.md]: relevant* !!!

**Review by turecross321 — CHANGES_REQUESTED** [review]
looks good now except for the relevant-typos 👍

File: patching/psp.md (hunk 9 above)
Comment (Beyley) [reply patching/psp.md]: that _is_ what that diff fixing yes

Comment (Beyley) [reply patching/psp.md]: nvm dyslexia strikes again

**Review by jvyden — APPROVED** [review]
LGTM, watch there be a severe grammar error that pops up immediately after merging

**Pull request merged**